src/js/components/profile-form/profile-form.html

```diff
@@ -501,7 +501,7 @@
 				<fieldset>
 					<legend><span translate>Children</span>
 						<button class="btn btn-info pull-right btn-add-child" ng-click="addChild()" type="button">
-							<span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Child
+							<span class="glyphicon glyphicon-plus" aria-hidden="true"></span> <span translate>Child</span>
```

Comment:
You can add the translate attribute directly to the span that has class="glyph...", no need for another span.